packages/@okta/vuepress-theme-prose/components/Breadcrumb.vue

```diff
@@ -2,11 +2,10 @@
 <div class="breadcrumb" v-if="showBreadcrumb">
   <div class="breadcrumb--container">
     <ol>
-      <li v-for="(crumb, index) in crumbs" :key="index">
-        <router-link v-if="crumb.link" :to="crumb.link">{{crumb.title}}</router-link>
```

Comment:
It seems like this isn't a link anymore?

Reply:
+1, breadcrumb items should still be links

Reply:
They were not actually links, just hardcoded links to the overview pages of every category and links to the pages themselves. Ok, I'll add overview links